src/mobu/dependencies/github.py

```diff
@@ -36,23 +55,38 @@ class CiManagerDependency:
     """
 
     def __init__(self) -> None:
-        self.ci_manager: CiManager
+        self._ci_manager: CiManager | None = None
+
+    @property
+    def ci_manager(self) -> CiManager:
+        if self._ci_manager is None:
+            raise RuntimeError("CiManager has not been initialized yet")
+        return self._ci_manager
 
     def __call__(self) -> CiManager:
         return self.ci_manager
 
     def initialize(
-        self, base_context: ContextDependency, users: list[User]
+        self,
+        base_context: ContextDependency,
+        users: list[User],
+        github_app_id: int,
+        github_private_key: str,
+        scopes: list[GafaelfawrScope],
```

Comment:
Nit: More than three parameters, so I would add `*` somewhere in the argument list to force all or some of them to be keyword arguments. It's hard for the human brain to keep track of this many positional arguments and what they all mean.